docs/DeveloperGuide.md

```diff
@@ -156,18 +156,18 @@ This section describes some noteworthy details on how certain features are imple
 
 ### Add Gender
 
-The Add Gender feature allows users to add a gender field (Male / Female) to a person in the contact list. It is performed as a part 
-of `addCommand#execute()`.
+The Add Gender feature allows users to add a gender field (Male / Female) to a person in the contact list. It is performed as a part
+ of `addCommand#execute()`.
```

Comment:
Since the sequence diagram uses `AddPersonCommand`, this line should be updated to match as well.

```suggestion
 of `addPersonCommand#execute()`.
```